can-validate.js

```diff
@@ -1,35 +1,6 @@
-var Construct = require('can-construct');
-var validate = require('can-validate/lib/validate');
-var helpers = require('can-validate/lib/helpers');
+var formatErrors = require('can-validate/format-errors/');
```

Comment:
don't use paths that end with `/` so other module loaders can load files.  Use `can-validate/format-errors/format-errors`.